src/EFCore.MySql/Storage/Internal/MySqlConnectionSettings.cs

```diff
@@ -35,7 +35,7 @@ public MySqlConnectionSettings(string connectionString)
             TreatTinyAsBoolean = csb.TreatTinyAsBoolean;
         }
 
-        public MySqlGuidFormat GuidFormat { get; }
+        public virtual MySqlGuidFormat GuidFormat { get; }
```

Comment:
This change is necessary to allow Moq to override the value returned from this property. Perhaps there are other ways to allow this, too.

Reply:
While you are at it, please make `TreatTinyAsBoolean` virtual as well. We should have made then both virtual in the first place.